tensorflow/lite/micro/micro_allocator.h

```diff
@@ -118,24 +117,21 @@ class MicroAllocator {
   // Note: Please use alignas(16) to make sure tensor_arena is 16
   // bytes aligned, otherwise some head room will be wasted.
   // TODO(b/157615197): Cleanup constructor + factory usage.
-  static MicroAllocator* Create(uint8_t* tensor_arena, size_t arena_size,
-                                ErrorReporter* error_reporter);
+  static MicroAllocator* Create(uint8_t* tensor_arena, size_t arena_size);
```

Comment:
MicroAllocator can be used external to the TFLM codebase (e.g. when we are using the multi-tenant APIs). It might be prudent to keep the backwards compatible ErrorReporter API.

Reply:
Done.